### PR TITLE
Mute Management Settings Page

### LINKED ISSUE #943: Mute management UI
**Is your feature request related to a problem? Please describe.**
There is no current way to see all the stackers that you've muted. In order to unmute a stacker, you have to find their profile, or an item from them to unmute.

**Describe the solution you'd like**
I'd like a page or a tab where you can see all the stackers you've muted in one place. I'd also like to be able to unmute them from this UI so it's very easy to manage your mutes in one central place.

One option is just on your own profile page, `mutes` could be a new dropdown selection, and the UI could render all of your muted stackers. This list should only work for yourself, i.e. you should not be able to see who _other_ stackers have muted.

Another option is somewhere on your settings page, though there's probably more UI work needed to fit this in. The same usability concerns from above apply here, too.

**Describe alternatives you've considered**
None

**Additional context**
This was recently discussed [here](https://stacker.news/items/472070/r/WeAreAllSatoshi?commentId=472355).

### PR DESCRIPTION
**Pull request by SatsAllDay**
## Description
Closes #943 

Add a muted users management tab to the settings page - a tab where you can view all the stackers whom you've muted, and you can manage your mutes for each individually. The approach is as follows:

1. Add a "muted users" tab to the settings page that links to new page at `/settings/mutes`
2. Create a new user graphql query which returns all users that you've muted
3. Displays a list of these users, in pages of 21
4. Each user in the list has an action overflow menu, where you can manage your subscription(s) and mutes, just like on the user profile page
5. When you unmute a stacker in the list, the list reloads to reflect that change, including removing said stacker from the list

## Screenshots
New "muted users" tab on settings page nav bar:
image

The muted users listing:
image

Pagination available:
image

No more data left to fetch
image

Mobile preview:
image

## Additional Context
This is largely a port/copy/paste from #1000 with some minor changes for mutes instead of subscriptions

## Checklist

- [X] Are your changes backwards compatible?

- [X] Did you QA this? Could we deploy this straight to production?

- [X] For frontend changes: Tested on mobile?

### CODE REVIEW

**Comment by SatsAllDay** [comment]
I changed the tab label to `muted stackers` but I was too lazy to take new screenshots :)

File: pages/settings/mutes/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export const getServerSideProps = getGetServerSideProps({ query: MY_MUTED_USERS, authRequired: true })
+
+export default function MuMutedUsers ({ ssrData }) {
+  const muteUserContextValue = useMemo(() => ({ refetchQueries: ['MyMutedUsers'] }), [])
+  return (
+    <Layout>
+      <div className='pb-3 w-100 mt-2'>
+        <SettingsHeader />
+        <div className='mb-4 text-muted'>Well now, reckon these here are the folks you've gone and silenced.</div>
```
[the remark this answers is not in the document]
Comment (SatsAllDay) [reply pages/settings/mutes/index.js:17]: lol coderabbit, please

**Comment by ekzyis** [comment]
> I changed the tab label to muted stackers but I was too lazy to take new screenshots :)

business idea right there: screenshot diff tool powered by AI

File: pages/settings/mutes/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ekzyis) [reply pages/settings/mutes/index.js:17]: this might be the best part about coderabbit, it's so innocent

**Review by ekzyis — APPROVED** [review]
Found nothing to complain about, well done!

**Pull request merged**